services/auth/src/api/routes/profile.py

```diff
@@ -0,0 +1,73 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from api.deps import AccessRequire, Session
+from models import User, Profile
+from schemas import MeResponse, ProfileSchema, ProfileUpdate
+
+router = APIRouter(prefix="/me", tags=["Profile"])
+
+
+@router.get("", response_model=MeResponse)
+async def get_me(
+    require: AccessRequire,
+    session: Session,
+) -> MeResponse:
+    """Get current user with profile."""
+    user_id = require.payload["sub"]
+    
+    # Load user with profile
+    result = await session.execute(
+        select(User).where(User.id == user_id)
+    )
+    user = result.scalar_one_or_none()
+    
+    if not user:
+        from fastapi import HTTPException, status
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="User not found",
+        )
+    
+    # Create profile if doesn't exist
+    if not user.profile:
+        profile = Profile(user_id=user.id)
+        session.add(profile)
+        await session.commit()
+        await session.refresh(user)
+    
+    return MeResponse(
+        id=user.id,
+        email=user.email,
+        is_verified=user.is_verified,
+        profile=ProfileSchema.model_validate(user.profile),
+    )
+
+
+@router.patch("", response_model=ProfileSchema)
+async def update_me(
+    data: ProfileUpdate,
+    require: AccessRequire,
+    session: Session,
+) -> ProfileSchema:
+    """Update current user profile."""
+    user_id = require.payload["sub"]
+    
+    # Get or create profile
+    profile = await session.scalar(
+        select(Profile).where(Profile.user_id == user_id)
+    )
+    
+    if profile is None:
+        profile = Profile(user_id=user_id)
+        session.add(profile)
+    
+    # Update fields
+    for field, value in data.model_dump(exclude_unset=True).items():
+        setattr(profile, field, value)
+    
+    await session.commit()
+    await session.refresh(profile)
+    
+    return ProfileSchema.model_validate(profile)
```

Comment:
Важный момент по `response_model=ProfileSchema` в декораторе в связке с `return ProfileSchema.model_validate(profile)`!

Такая связка валидирует данные дважды, то есть дважды производится трансформация из одного типа в дргугой проверяя одно другим.

Рекомендация использовать `response_model=ProfileSchema` без `return ProfileSchema.model_validate(profile)`. Вместо этого вернуть объект модели базы данных напрямую, то есть просто `return profile`. Так будет проходить один раз та конвертация под капотом. Главное чтобы `ProfileSchema` имела в себе `model_config = ConfigDict(from_attributes=True)`. Типизация функции соответсвенно будет равняться модели базы данных.
```suggestion
    return profile
```